Backend/model/model.py

```diff
@@ -85,6 +138,44 @@ class Donor(BaseModel):
 class Child(BaseModel):
     childName: str
     motherName: str
-    child_age: str
+    child_age: str = Field(..., min_length=1, max_length=2)
     gender: str
     isActive: bool
+
+    @validator("childName")
+    def child_name_validation(cls, childName):
```

Comment:
Probably we can reuse this name validation function for all models and call it where ever it is required instead of repeating for all names again and again with different method names.